File: Parcial 1/Clases/Clase 19-05/Ejemplo_Pila.py
pila = []

pila.append(10) # Insertar elemento 10 en la pila
pila.append(20) # Insertar elemento 20 en la pila
pila.append(30) # Insertar elemento 30 en la pila

while len(pila) > 0:
    print(pila[-1], end=" ") # Imprimir elemento superior de la pila
    pila.pop() # Eliminar elemento superior de la pila

File: Parcial 1/Clases/Clase 19-05/Ejemplo_Pila.cpp
#include <iostream>
#include <stack>

int main() {
    std::stack<int> pila;

    pila.push(10); // Insertar elemento 10 en la pila
    pila.push(20); // Insertar elemento 20 en la pila
    pila.push(30); // Insertar elemento 30 en la pila

    while (!pila.empty()) {
    std::cout << pila.top() << " "; // Imprimir elemento superior de la pila
    pila.pop(); // Eliminar elemento superior de la pila
    }
    
    return 0;
}
